hooks/useMap.tsx

```diff
@@ -27,6 +27,10 @@ const useMap = () => {
       setIsVideoOpen(!isVideoOpen)
       return
     }
+    if (area.id === 'live-show') {
+      window.open('https://xcelencia.world', '_blank')
```

Comment:
window.open works across most modern browsers, but some browsers might block pop-ups depending on user settings. Ensure the function is triggered within an event handler directly to improve compatibility. This can help avoid pop-up blockers on some browsers.